src/Shared/Constants.cs

```diff
@@ -60,8 +60,6 @@ internal static class MSBuildConstants
         /// </summary>
         internal const string CurrentToolsVersion = "Current";
 
-        // if you change the key also change the following clones
-        // Microsoft.Build.OpportunisticIntern.BucketedPrioritizedStringList.TryIntern
```

Comment:
I'd leave the comment and update it with the new location

Reply:
Ah, comment no longer relevant

---
In reply to: [486044656](https://github.com/dotnet/msbuild/pull/5663#discussion_r486044656) [](ancestors = 486044656)